### DISCUSSION TITLE
Ship stuck on Al Hurr

### DISCUSSION THREAD

**Discussion by Vemenous-Repentile** [endless-sky/endless-sky#10268] [Q&A]
I'm in the late game (or so I thought?) and I've just had a ship get stuck in an undiscovered system - Al Hurr. 

Can anyone give me a clue how to trigger the relevant mission chain I assume exists that will reveal the system so I can get my lovely heavy, jump drive enabled, robot battleship back, please? 

I miss them. 

:-(

**Comment by TheGiraffe3** [comment]
Not sure what mission chain it is, but you can edit your save file and put the ship in the system you are in.

**Comment by eebop** [comment]
Going through undiscovered systems is a known bug, and al hurr is from hai reveal, which is broken right now. Just edit it, unfortunately

**Reply by TomGoodIdea** [reply]
But this bug was supposed to be fixed in #6127, which was part of 0.10.5

**Reply by TheGiraffe3** [reply]
Supposed to. 

I am guessing this will be a recurring bug.
